src/App.js

```diff
@@ -1,22 +1,36 @@
 import React from "react";
-import "./App.css";
-import ToggleSwitch from "./components/ToggleSwitch";
+import styled from "styled-components";
+import SettingsBox from "./components/SettingsBox";
+import Head from "./components/Head";
+import defaultSettings from "./settings";
 
-function App() {
-  const [settings, setSettings] = React.useState({
-    darkmode: false
-  });
+const Playground = styled.div`
+  height: 100vh;
+  width: 100vw;
+  background-color: #282c34;
+`;
 
-  function handleToggleDarkmode(value) {
-    setSettings({
-      darkmode: value
-    });
+const Grid = styled.div`
+  height: fit-content;
+  padding: 20px;
+  display: grid;
+  grid-gap: 20px;
+  grid-template-columns: repeat(auto-fit, minmax(205px, 1fr));
+  > * {
```

Comment:
This looks weird. Is this common practice?

Reply:
Yes, that's a way to get the columns to wrap content, effectively replacing `grid-template-rows`. Might get reworked once the grid will be more populated